src/systems/audio-debug-system.js

```diff
@@ -143,7 +150,7 @@ AFRAME.registerSystem("audio-debug", {
   },
 
   enableDebugMode(enabled, force = false) {
-    if ((enabled === undefined || enabled === this.data.enabled) && !force) return;
+    if (((enabled === undefined || enabled === this.data.enabled) && !force) ||   this.unsupported) return;
```

Comment:
Nit: This line seems to have extra spaces.

```suggestion
    if (((enabled === undefined || enabled === this.data.enabled) && !force) || this.unsupported) return;
```